bindata/cronjob-scheduler/config/configmap.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cron-scheduler-configmap
```

Comment:
Maybe we should make sure this is associated with whereabouts, maybe...

> whereabouts-reconciler-cron-scheduler-configmap

Reply:
I like your suggestion. I'm going with `whereabouts-reconciler-scheduler-configmap`. Not exactly a name that rolls off the tongue but it'll do.

Reply:
Yep, sounds good, let's update this one!